src/payment/create-payment-strategy-registry.ts

```diff
@@ -341,6 +342,25 @@ export default function createPaymentStrategyRegistry(
         )
     );
 
+    registry.register(PaymentStrategyType.CHECKOUTCOM_GOOGLE_PAY, () => {
+        const initializer = new GooglePayCheckoutcomInitializer();
+
+        return new CheckoutcomPaymentStrategy(
+            store,
+            checkoutActionCreator,
+            paymentMethodActionCreator,
+            paymentStrategyActionCreator,
+            paymentActionCreator,
+            orderActionCreator,
+            createGooglePayPaymentProcessor(
+                store,
+                initializer
+            ),
+            initializer
```

Comment:
Because the current googlepay-payment-strategy that this strategy extends doesn't expose the initial data through its flow, and this needs a pair of fields from the initialization

Reply:
Found another way to get the info i need, removed second initializerargument